packages/fluentui/react-northstar/src/components/Animation/useAnimationStyles.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  UseStylesResult,
+  ComponentAnimationProp,
+  unstable_getStyles as getStyles,
+  unstable_createAnimationStyles as createAnimationStyles,
+} from '@fluentui/react-bindings';
+import { ThemePrepared } from '@fluentui/styles';
+import * as React from 'react';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+import * as _ from 'lodash';
+import { AnimationProps } from './Animation';
+
+type UseAnimationStylesResult = {
+  classes: UseStylesResult['classes'];
```

Comment:
We can result only `root` there, should we just return `className` to simplify?

Reply:
changed